lib/notifications/client.rb

```diff
@@ -45,7 +45,7 @@ def send_sms(args)
     # @return [Notification]
     def get_notification(id)
       Notification.new(
-        speaker.send(:get, id)["notification"]
+        speaker.send(:get, id)["data"]["notification"]
       )
     end
```

Comment:
we don't indent to roll out any breaking changes to the public API (hopefully ever), so I wouldn't worry about being too defensive - however, if the service returns a non-200 (eg 404, 500), it won't contain `["data"]`, instead it will contain the error struct, which we'll definitely want to bubble up to user apps in a graceful manner